### DISCUSSION TITLE
SPM model to simulate degradation

### DISCUSSION THREAD

**Discussion by iurkens** [pybamm-team/PyBaMM#2517] [Q&A]
Hi

I want to (qualitatively) simulate the impact of additional discharges due to Vehicle-to-grid. I've used the following model configuration with the Okane2022 parameter set:

Model: SPM
options ={
"SEI":"solvent-diffusion limited", 
"SEI film resistance": "distributed", 
"SEI porosity change": "true",
"SEI on cracks": "true",

"particle mechanics": ("swelling and cracking", "swelling only"),
"lithium plating": "partially reversible",
"lithium plating porosity change": "true",
"loss of active material" : "stress and reaction-driven"
}

Yet, there is no additional degradation due to higher DoDs, and i think that cannot be true with particle cracking enabled. 

My question:
In the paper of (OKane et al.  2022) these submodels were used in combination with the DFN model. Do I need to use the DFN model to successfully simulate the impact of DoD on battery degradation?

With kind regards

**Comment by iurkens** [comment]
Edit: 

1. I've concluded that using the DFN model doesn't help, however, I was only to simulate it on smaller timescales (1 year).
2. I think LAM "stress-driven" would give the same results because there are not enough parameters in the Okane set to simulate the "reaction-driven" part.

If someone could give me some advice, I would be very grateful.
![image](https://user-images.githubusercontent.com/116073555/205607944-a16de06a-3775-45a2-ab69-04ecf801d5ba.png)

These were the results with a variation of DoDs (34%, 48% and 61%) during 10 years
![image](https://user-images.githubusercontent.com/116073555/205608430-9974b269-fdb1-450f-b38b-4c92f48af560.png)


With kind regards

**Comment by DarioSlaifsteinSk** [comment]
Hi @iurkens,
I'm having the same issue! I can't find how the `SEIgrowth` submodel relates to the current.
